benchmarks/user/advisor/README.md

```diff
@@ -10,6 +10,9 @@ After the run has finished you should be able to plot a roofline with the result
 To export roofline data to JSON, use:
 * `python3 advisor_to_json.py --name Roofline --project <advisor-project-name>`
 
+To create a read-only snapshot for use with Intel Advisor GUI, use:
+* `advixe-cl --snapshot --project-dir=<advisor-project-name> pack -- /<new-snapshot-name>`
```

Comment:
what's a snapshot?

Reply:
Snapshot is this small in size file used to save the results and see them efficiently in GUI.
https://software.intel.com/content/www/us/en/develop/documentation/advisor-user-guide/top/results/creating-a-read-only-result-snapshot.html